src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Controllers/RolesController.cs

```diff
@@ -36,7 +37,7 @@ public ActionResult Index()
         {
             _authService.CheckAccess(AuthOperations.AdminDashboard.View);
 
-            var roles = _rolesService.GetAllRoles().ConvertAll(r => new RoleViewModel(r));
+            var roles = _rolesService.GetAllRoles().ToList().ConvertAll(r => new RoleViewModel(r));
```

Comment:
ToList?

Reply:
the same thing, _rolesService is IColection

Reply:
So what?